### DISCUSSION TITLE
`experimental Nix feature 'nix-command' is disabled`

### DISCUSSION THREAD

**Discussion by NeoFax75** [juspay/nix-dev-home#53] [Q&A]
On a brand new install, I get the following error:

```error: experimental Nix feature 'nix-command' is disabled; use '--extra-experimental-features nix-command' to override```

**Comment by srid** [comment]
> a brand new install,

Did you install Nix using the link in [the README](https://github.com/juspay/nix-dev-home?tab=readme-ov-file#getting-started)?

**Reply by NeoFax75** [reply]
No, from the official webpage.  I will reinstall after work, and try again.
